gbfs.md

```diff
@@ -1009,6 +1009,7 @@ Field Name | REQUIRED | Type | Defines
 &emsp;&emsp;&emsp;\-&nbsp;`ride_allowed` | Conditionally REQUIRED | Boolean | REQUIRED if `rules` array is defined. Is the undocked (“free bike”) ride allowed to start and end in this zone? <br /><br /> `true` - Undocked (“free bike”) ride can start and end in this zone. <br /> `false` - Undocked (“free bike”) ride cannot start and end in this zone.
 &emsp;&emsp;&emsp;\-&nbsp;`ride_through_allowed` | Conditionally REQUIRED | Boolean | REQUIRED if `rules` array is defined. Is the ride allowed to travel through this zone? <br /><br /> `true` - Ride can travel through this zone. <br /> `false` - Ride cannot travel through this zone.
 &emsp;&emsp;&emsp;\-&nbsp;`maximum_speed_kph` | OPTIONAL | Non-negative Integer | What is the maximum speed allowed, in kilometers per hour? <br /><br /> If there is no maximum speed to observe, this can be omitted.
+&emsp;&emsp;&emsp;\-&nbsp;`station_parking` | OPTIONAL | Boolean | Vehicle MUST be parked at stations defined in station_information.json within this geofence zone? <br /><br />`true` - MUST only be parked at stations.  <br /> `false` - MAY be parked outside of stations.
```

Comment:
Usage og capitalized "MUST" and "MAY" seems non-standard. These capitalized keywords are used for describing the spec, and not for the semantics of the data itself. 

I suggest this:

Can the vehicle only be parked at stations defined in station_information.json within this geofence zone? <br /><br />`true` - Can only be parked at stations. <br /> `false` - May be parked outside of stations.

Reply:
Thanks for catching this, @testower . We will make sure it is corrected before merging. @viestat - could you update the language?